File: primer_trimestre/poo/java/proyectos_java/ejercicio_1_java/src/Employee.java
    public class Employee{
    private String name;
    private double baseSalary=1000000;


    public String setName(String name) {
        return this.name = name;
    }
    public double setBaseSalary(double baseSalary) {
       return this.baseSalary = baseSalary;

    }
    public String getName() {
        return name;
    }
    public int getBaseSalary() {
        return (int) this.baseSalary;
    }
    }

File: primer_trimestre/poo/php/ejercicios_php/first/libreria/Employee.php
<?php
class Employee {
    private $name;
    private $baseSalary = 1000000;

    public function setName($name) {
        $this->name = $name;
        return $this->name;
    }

    public function setBaseSalary($baseSalary) {
        $this->baseSalary = $baseSalary;
        return $this->baseSalary;
    }

    public function getName() {
        return $this->name;
    }

    public function getBaseSalary() {
        return (int) $this->baseSalary;
    }
}
?>